db/migrate/20190117015241_create_users.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :users do |t|
       t.string :name
       t.string :email
-      t.boolean :gender
+      t.integer :gender
```

Comment:
giá trị default mô rồi, và ko cho phép null nữa nhé
`tìm hiểu lại enum in rails ` đi